server/src/main/java/com/genymobile/scrcpy/ScreenEncoder.java

```diff
@@ -50,6 +50,12 @@ public ScreenEncoder(Device device, Streamer streamer, int videoBitRate, int max
         this.downsizeOnError = downsizeOnError;
     }
 
+    @Override
+    public void onFoldChanged(int displayId) {
+        // TODO: rename rotationChanged to something like displayChanged? To discuss with @rom1v
+        rotationChanged.set(true);
```

Comment:
@rom1v Here I guess this variable could be renamed? Or do we need two variables?